### PR TITLE
Changing beta labels to updated ones

### LINKED ISSUE #13: Update to new node labels for Kubernetes 1.16
The existing beta-type node labels are deprecated in Kubernetes 1.16, as documented in this KEP: https://github.com/kubernetes/website/pull/17498/files

Ensure that the new labels apply as required and update the README.

### PR DESCRIPTION
**Pull request by davidjumani**
Fixes #13

### CODE REVIEW

**Comment by davidjumani** [comment]
@onitake Looks good ?

File: README.md [hunk 1]
```diff
@@ -79,9 +79,8 @@ This will treat the node as 'uninitialized' and cause the CCM to apply metadata
 
 Supported labels are:
 * kubernetes.io/hostname (= the instance name)
-* beta.kubernetes.io/instance-type (= the compute offering)
-* failure-domain.beta.kubernetes.io/zone (= the zone)
-* failure-domain.beta.kubernetes.io/region (also = the zone)
+* node.kubernetes.io/instance-type (= the compute offering)
+* topology.kubernetes.io/region (= the zone)
```
Comment (onitake) [inline README.md:84]: I think `topology.kubernetes.io/zone` still exists, according to the KEP.

Maybe we should also keep the old labels in the documentation, with a note like this: "(up to Kubernetes 1.16)"
The new labels should have: "(Kubernetes 1.17 and later)"

**Review by onitake — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (davidjumani) [reply README.md]: @onitake Changes made

**Comment by rhtyd** [comment]
@onitake @joschi36 are you lgtm on merging this?

**Review by joschi36 — APPROVED** [review]

**Comment by joschi36** [comment]
LGTM

**Pull request merged**